lib/grammers-client/src/client/files.rs

```diff
@@ -264,14 +264,10 @@ impl Client {
                             );
                         }
                         Err(InvocationError::Rpc(err)) => {
-                            // Retry on rate limit
-                            if err.code == 420 {
```

Comment:
Was it intended to remove this retry logic in this PR?

Reply:
yes, the wait login is inside https://github.com/Lonami/grammers/blob/master/lib/grammers-client/src/client/net.rs#L251-L267

Reply:
Yes but this change is still not directly related to handling `FILE_MIGRATE` which is what I meant (if this change was for this or another PR).

Reply:
I separated it into another PR: https://github.com/Lonami/grammers/pull/149